### PR TITLE
feat: exists

### PR DESCRIPTION
**Pull request by onhate**
Feature to check if exists a context created.

Issue: Today I do get within try/catch but it still logs an error to the console when in production mode, I want an easy way to check if the context exists without logging to the error console.

```js
try { return !!Context.get() } catch(e) { return false }
```

to

```js
Context.exists()
```

### CODE REVIEW

**Comment by onhate** [comment]
@odedglas

**Comment by odeadglaz** [comment]
Looks solid, thanks for your contribution (And for the README spelling :))

File: src/ExecutionContext/index.js [hunk 1]
```diff
@@ -40,6 +40,14 @@ class ExecutionContext {
         this.manager.update(context);
     }
 
+    /**
+     * Check if the current asynchronous execution context exists.
+     * @return {*}
```
Comment (odeadglaz) [inline src/ExecutionContext/index.js:45]:
```suggestion
     * @return {boolean}
```

**Review by odeadglaz — APPROVED** [review]

**Comment by onhate** [comment]
Thanks @odeadglaz!

**Pull request merged**